pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -46,7 +46,7 @@ func Bootstrap(bootstrapConfiguration deployments.BootstrapConfiguration, target
 	coreBootstrapDone := false
 
 	if clientSet, err := kubernetes.GetAdminClientSet(); err == nil {
-		_, err := clientSet.ServerVersion()
+		_, err := clientSet.Discovery().ServerVersion()
```

Comment:
Why change this?

Reply:
In internal/pkg/skuba/kubernetes/clientset.go, return value was changed from *clientset.Clientset to clientset.Interface. I could find more info: https://github.com/kubernetes/client-go/blob/52092c3c67fa52f5d3f63c03f441560344cce2eb/kubernetes/clientset.go#L352